src/styles/app.css

```diff
@@ -0,0 +1,12 @@
+.App {
+  display: flex;
```

Comment:
Theres a bit of a UX issue when it comes to entering a value that doesnt return any results here. If you provide a nonsense value the bottom half of the screen show white, and the test 'no results' is barely visible with the contrast it has to the background. Theres a couple of things you could do here to improve it:
- add a property of `min-height: 100vh` to the `app` class. ensure you have a solid background for the whole app,
- add a background and some border styling to the `p` tag containing the 'no results' text (best practice would be to add a class to the tag rather than targeting the `<p>` element itself)
- centre the `p` tag so it fits more inline with the styling of the rest of the app